app/controllers/site_controller.rb

```diff
@@ -106,6 +106,11 @@ def about
     @locale = params[:about_locale] || I18n.locale
   end
 
+  def communities
+    @locale = params[:communities_locale] || I18n.locale
```

Comment:
Is language the correct way to be selecting a community? Shouldn't it be location driven?